### PR TITLE
Install mto on a multiarch cluster to run e2e on it

### PR DESCRIPTION
**Pull request by lwan-wanglin**
for [MULTIARCH-4313](https://issues.redhat.com/browse/MULTIARCH-4313) [MULTIARCH-4575](https://issues.redhat.com/browse/MULTIARCH-4575)

- Adding a profile on multiarch cluster with multiarch-tuning-operator installed and run e2e on it to see if therere potential issues
- Now we deploy it using the latest bundle images in registry.ci, and will change to via operatorhub when it's ready.

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-azure-ipi-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-azure-ipi-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-gcp-ipi-ovn-ipsec-mto-arm-mixarch-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-azure-ipi-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-aws-ipi-disc-priv-sts-efs-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-aws-ipi-disc-priv-sts-efs-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-azure-ipi-mto-arm-mixarch-f7

**Comment by lwan-wanglin** [comment]
the operator has installed on multiarch cluster and start to run e2e on it in above job, @aleskandro @prashanth684 could you help to review?thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 1]
```diff
@@ -769,6 +773,25 @@ tests:
...
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4s_v3
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: mto-deploy
```
Comment (aleskandro) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:792]:
```suggestion
    - ref: mto-deploy
    - ref: ipi-install-heterogeneous
```

I'd have the operator deployed before the cluster becomes heterogeneous in compute nodes arch. It shouldn't change here, but it is what we are going to recommend the users to do.

File: ci-operator/step-registry/mto/deploy/mto-deploy-commands.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+apiVersion: multiarch.openshift.io/v1alpha1
+spec:
+  logVerbosity: Normal
+metadata:
+  name: cluster
+EOF
+oc wait pods -n ${NAMESPACE} \
+  -l controller=pod-placement-controller \
+  --for=condition=Ready=True
+oc wait pods -n ${NAMESPACE} \
```
[the remark this answers is not in the document]
Comment (lwan-wanglin) [reply ci-operator/step-registry/mto/deploy/mto-deploy-commands.sh:30]: sure, will fix it

**Comment by lwan-wanglin** [comment]
@aleskandro PTAL again,thanks!

**Review by aleskandro — COMMENTED** [review]
Thanks @lwan-wanglin 

/lgtm

**Comment by lwan-wanglin** [comment]
cc @liangxia to review,thanks!

**Pull request merged**